docs/src/modules/components/PageContext.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const PageContext = React.createContext();
```

Comment:
That's missing the previous default values. They should be present for documentation purposes.

Reply:
Maybe a comment in this case?

Reply:
Documentation gets outdated. With actual values vscode is able to acquire type information and enables autocomplete.

Reply:
> Documentation gets outdated. 

Do does default values that are never used.

> With actual values vscode is able to acquire type information and enables autocomplete.

Ok, that's the why. I'm adding them back.

Reply:
I believe that this is actual the typescript language server so any editor (plugin) that is leveraging that benefits. I just like my autocomplete ;)